packages/shared/routes/setup/import/views/Success.svelte

```diff
@@ -20,7 +20,7 @@
     }
 </script>
 
-<OnboardingLayout onBackClick={handleBackClick}>
+<OnboardingLayout allowBack={false}>
```

Comment:
You should remove the `handleBackClick()` function as well if we don't allow the user to go back.